src/panels/lovelace/cards/hui-history-graph-card.ts

```diff
@@ -164,7 +164,9 @@ export class HuiHistoryGraphCard extends LitElement implements LovelaceCard {
 
     await this._fetchStatistics(sensorNumericDeviceClasses);
 
-    this._setRedrawTimer();
+    if (this.isConnected) {
```

Comment:
Can we move this check inside the `_setRedrawTimer` function, and return after clearing the timer?

Reply:
Actually now I'm wondering if there's a similar issue around `await getSensorNumericDeviceClasses(this.hass!)` on line 136.

We can disconnect while we are awaiting there, and then we will set this._subscribed while disconnected when the await returns. Then we will not set the redraw interval.

Then if we reconnect, we will skip subscribe as we are already subscribed. And we will not set the redraw interval.